config/patroni/patroni-A/99_node_A.yml

```diff
@@ -0,0 +1,9 @@
+scope: alfdb
```

Comment:
file is no longer used? so can be removed?

Reply:
is needed, contains cluster and node name

Reply:
maybe I'm missing something? but I don't see any reference to this file in the docker-compose.yml?
instead the patroni cluster and node name are defined as environment variables

Reply:
you are correct, I already forgot I switched to environment variables